<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>1061</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget{
background-color: #d63384; 
}
#label {
    color: white;
    font-size: 18px;
    font-weight: bold;
    padding-left: 15px;
}

/* Exit button */
#exitButton {
    background-color: ;
	background-color: rgb(255, 190, 111);
    color: white;
    font-size: 16px;
    border-radius: 6px;
}

#exitButton:hover {
    background-color: #a61e5c;
    border-radius: 6px;
}

</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Healthcare Appointment System&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>20</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>1061</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 190, 111);</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">

    color:rgb(205, 27, 124);
    font-size: 18px;
    font-weight: bold;          /* ðŸ‘ˆ THIS LINE */
    selection-color: white; </string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:15pt;&quot;&gt;Login Page&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>150</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(214, 51, 132);
border-radius: 6px;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>249</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(214, 51, 132);
border-radius: 6px;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usernameLineEdit">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>150</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="passwordLineEdit">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>250</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="roleComboBox">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>340</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(214, 51, 132);
border-radius: 6px;
font-weight: bold;
font-size: 16px;</string>
      </property>
      <item>
       <property name="text">
        <string>Patient</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Doctor</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>340</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(214, 51, 132);
border-radius: 6px;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Role&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>440</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#loginButton {
    border-color: rgb(255, 255, 255);
    background-color: rgb(214, 51, 132);
    color: white;
    font-size: 18px;
    font-weight: bold;          /* ðŸ‘ˆ THIS LINE */
    border-radius: 6px;
    border: 1px solid rgb(255, 255, 255);
    selection-color: white; 
}

#loginButton:hover {
    background-color: rgb(255, 190, 111);
    border-radius: 6px;
}
</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="patientDashboardLabel">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">

    color:rgb(205, 27, 124);
    font-size: 18px;
    font-weight: bold;          /* ðŸ‘ˆ THIS LINE */
    selection-color: white; </string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Patient Dashboard&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bookAppointmentButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>150</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#bookAppointmentButton {
    border-color: rgb(255, 255, 255);
    background-color: rgb(214, 51, 132);
    color: white;
    font-size: 18px;
    font-weight: bold;          /* ðŸ‘ˆ THIS LINE */
    border-radius: 6px;
    border: 1px solid rgb(255, 255, 255);
    selection-color: white; 
}

#bookAppointmentButton:hover {
    background-color: rgb(255, 190, 111);
    border-radius: 6px;
}</string>
      </property>
      <property name="text">
       <string>Book Appointment</string>
      </property>
     </widget>
     <widget class="QPushButton" name="viewAppointmentsButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>250</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#viewAppointmentsButton {
    border-color: rgb(255, 255, 255);
    background-color: rgb(214, 51, 132);
    color: white;
    font-size: 18px;
    font-weight: bold;          /* ðŸ‘ˆ THIS LINE */
    border-radius: 6px;
    border: 1px solid rgb(255, 255, 255);
    selection-color: white; 
}

#viewAppointmentsButton:hover {
    background-color: rgb(255, 190, 111);
    border-radius: 6px;
}</string>
      </property>
      <property name="text">
       <string>My Appointments</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton0">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>350</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#backButton0 {
    border-color: rgb(255, 255, 255);
    background-color: rgb(214, 51, 132);
    color: white;
    font-size: 18px;
    font-weight: bold;          /* ðŸ‘ˆ THIS LINE */
    border-radius: 6px;
    border: 1px solid rgb(255, 255, 255);
    selection-color: white; 
}

#backButton0:hover {
    background-color: rgb(255, 190, 111);
    border-radius: 6px;
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>30</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">

    color:rgb(205, 27, 124);
    font-size: 18px;
    font-weight: bold;          /* ðŸ‘ˆ THIS LINE */
    selection-color: white; </string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Book Appointment&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(214, 51, 132);
border-radius: 6px;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Select Doctor&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboDoctor">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>164</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#comboDoctor {
    border-color: rgb(255, 255, 255);
    background-color: rgb(214, 51, 132);
    color: white;
    font-size: 16px;
    font-weight: bold;          /* ðŸ‘ˆ THIS LINE */
    border-radius: 6px;
    border: 1px solid rgb(255, 255, 255);
    selection-color: white; 
}
</string>
      </property>
      <item>
       <property name="text">
        <string>Dr. Kumar (Cardiology)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dr. Meena (General)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dr. Ravi (Orthopedic)</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(214, 51, 132);
border-radius: 6px;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Select Date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(214, 51, 132);
border-radius: 6px;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Select Time&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>260</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>330</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmAppointmentButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>430</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#confirmAppointmentButton {
    border-color: rgb(255, 255, 255);
    background-color: rgb(214, 51, 132);
    color: white;
    font-size: 18px;
    font-weight: bold;          /* ðŸ‘ˆ THIS LINE */
    border-radius: 6px;
    border: 1px solid rgb(255, 255, 255);
    selection-color: white; 
}

#confirmAppointmentButton:hover {
    background-color: rgb(255, 190, 111);
    border-radius: 6px;
}</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backButton1">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>430</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#backButton1 {
    border-color: rgb(255, 255, 255);
    background-color: rgb(214, 51, 132);
    color: white;
    font-size: 18px;
    font-weight: bold;          /* ðŸ‘ˆ THIS LINE */
    border-radius: 6px;
    border: 1px solid rgb(255, 255, 255);
    selection-color: white; 
}

#backButton1:hover {
    background-color: rgb(255, 190, 111);
    border-radius: 6px;
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="Title">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">

    color:rgb(205, 27, 124);
    font-size: 20px;
    font-weight: bold;          /* ðŸ‘ˆ THIS LINE */
    selection-color: white; </string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:15pt;&quot;&gt;Appointments&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="appointmentsTable">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>101</y>
        <width>461</width>
        <height>271</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Doctor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="backButton2">
      <property name="geometry">
       <rect>
        <x>288</x>
        <y>420</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#backButton2 {
    border-color: rgb(255, 255, 255);
    background-color: rgb(214, 51, 132);
    color: white;
    font-size: 18px;
    font-weight: bold;          /* ðŸ‘ˆ THIS LINE */
    border-radius: 6px;
    border: 1px solid rgb(255, 255, 255);
    selection-color: white; 
}

#backButton2:hover {
    background-color: rgb(255, 190, 111);
    border-radius: 6px;
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logoutButton1">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>420</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#logoutButton1 {
    border-color: rgb(255, 255, 255);
    background-color: rgb(214, 51, 132);
    color: white;
    font-size: 18px;
    font-weight: bold;          /* ðŸ‘ˆ THIS LINE */
    border-radius: 6px;
    border: 1px solid rgb(255, 255, 255);
    selection-color: white; 
}

#logoutButton1:hover {
    background-color: rgb(255, 190, 111);
    border-radius: 6px;
}</string>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
